### PR TITLE
UI should reflect read only status

### PR DESCRIPTION
**Pull request by sviatosluv**

### CODE REVIEW

File: lib/chat_web/live/main_live/index.html.heex [hunk 1]
```diff
@@ -699,6 +701,7 @@
                 type="room"
                 room_identity={@room_identity}
               />
+              <Layout.Popup.restrict_write_actions />
```
Comment (sergey-lukianov) [inline lib/chat_web/live/main_live/index.html.heex:704]: Do we need duplicating popups?

File: lib/chat_web/live/main_live/layout/message_input.ex [hunk 2]
```diff
@@ -25,11 +31,7 @@ defmodule ChatWeb.MainLive.Layout.MessageInput do
           id="dialog-form"
           class="basis-[99%] flex items-center justify-between"
           phx-change={JS.dispatch("chat:set-input-size", to: "#dialog-input")}
-          phx-submit={
-            JS.push("dialog/text-message")
-            |> JS.dispatch("chat:clear-value", to: "#dialog-input")
-            |> JS.dispatch("chat:set-input-size", to: "#dialog-input")
-          }
+          phx-submit={submit_dialog_text_message(%{writable: @writable})}
```
Comment (sergey-lukianov) [inline lib/chat_web/live/main_live/layout/message_input.ex:34]: It would be great to have some color indication as well

**Review by sergey-lukianov — CHANGES_REQUESTED** [review]
It is for message input

How about
- uploader
- room creation
- dialog opening
- room invites
- room request approval

File: lib/chat_web/live/main_live/layout/message_input.ex (hunk 2 above)
Comment (sergey-lukianov) [reply lib/chat_web/live/main_live/layout/message_input.ex:34]: I was about text-inputs and buttons
Popup was good enough. Now it is too scary. Text is following

### Read only mode
The device switched into read only mode. Storage drive needs to be upgraded.

File: lib/chat_web/live/main_live/index.html.heex [hunk 3]
```diff
@@ -739,7 +749,11 @@
                       <button
                         class="mr-4 flex items-center"
                         phx-click={
-                          if @room_requests == [], do: nil, else: show_modal("room-request-list")
+                          cond do
+                            @db_status.writable == :no -> show_modal("restrict-write-actions")
+                            @room_requests == [] -> nil
+                            true -> show_modal("room-request-list")
+                          end
```
Comment (sergey-lukianov) [inline lib/chat_web/live/main_live/index.html.heex:756]: we need to figure out another way to do this
This becomes too much of logic

Comment (sergey-lukianov) [reply lib/chat_web/live/main_live/index.html.heex:756]: might be creating `<.room_request_button ...>`

File: lib/chat_web/live/main_live/layout/message_input.ex (hunk 2 above)
Comment (sviatosluv) [reply lib/chat_web/live/main_live/layout/message_input.ex:36]: Screenshot 2023-02-02 at 21 46 52

File: lib/chat_web/live/main_live/index.html.heex (hunk 3 above)
Comment (sviatosluv) [reply lib/chat_web/live/main_live/index.html.heex]: done

**Review by sergey-lukianov — APPROVED** [review]

**Pull request merged**